chain/network/src/routing.rs

```diff
@@ -598,7 +598,6 @@ impl RoutingTable {
         }
     }
 
-    #[cfg(feature = "test_features")]
```

Comment:
Could you explain what happened here? Is `remove_edges` supposed to be a function that is only used for testing or not?

Reply:
@bowenwang1996 In the `try_save_edges(...)` method we have been removing edges from `self.edges_info` using `filter_map`. That is not correct, because we also need to remove edges from `self.raw_graph`.
Now, we remove those edges using `remove_edges` method. It was used previously for testing only, now it's used in production.